app/controllers/concerns/enlistment_filters.rb

```diff
@@ -76,8 +76,33 @@ def build_code_location
   def project_has_code_location?
     return unless CodeLocationSubscription.code_location_exists?(@project.id, @code_location.url,
                                                                  @code_location.branch, code_location_params[:scm_type])
-
-    flash[:notice] = t('.notice', url: @code_location.url, module_branch_name: @code_location.branch)
+    manage_deleted_enlistment
```

Comment:
I guess we should move this responsibility to Fisbot Api.

Reply:
The FISbot should not know anything about projects or enlistments or analyses, etc. It is only concerned with Code Locations and determining whether or not a Code Location needs to be updated.